go.graphics.swing/src/go/graphics/swing/AreaContainer.java

```diff
@@ -26,14 +26,14 @@
 
 import javax.swing.JPanel;
 
-import com.jogamp.opengl.GL2;
-import com.jogamp.opengl.GLAutoDrawable;
-import com.jogamp.opengl.GLCapabilities;
-import com.jogamp.opengl.GLEventListener;
-import com.jogamp.opengl.GLProfile;
-import com.jogamp.opengl.awt.GLCanvas;
-import com.jogamp.opengl.awt.GLJPanel;
-import com.jogamp.opengl.glu.GLU;
+import javax.media.opengl.GL2;
+import javax.media.opengl.GLAutoDrawable;
```

Comment:
There is, yes. The desired version now uses javax.media.opengl as it has been standardised.

Unless there's a reason to revert tot he old version, you need to update packages to match the version that's in build.gradle.